conanfile.py

```diff
@@ -5,11 +5,11 @@
 
 class PahocConan(ConanFile):
     name = "Ne10"
-    version = "1.2.2-2018.11.15" # version number rarely changes, so add date
-    #source_subfolder = "sources"
+    version = "1.2.2-2018.11.15"  # version number rarely changes, so add date
```

Comment:
Should we bump the version number to 1.2.2-2020.02.11 so dependencies pick up the correct version?

Reply:
Sure, we can do that. I tested and it builds. Should I add it to this PR?

Reply:
Yes please. Could you then also add the new version number to libMOT?

Reply:
Ok pushed. Sure I'll change libMOT as well.